tests/test_commands.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+
+import unittest
+
+from click.testing import CliRunner
+from syncano_cli.main import cli
+
+
+class BaseCommandsTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.runner = CliRunner()
+
+
+class SyncCommandsTest(BaseCommandsTest):
+    # TODO: make test more 'real';
```

Comment:
It may be better to either create an issue/task or whatever for this rather than TODO comment unless it is meant to be solved soon